Examples/CaseStudies/SwiftUICaseStudies/00-RootView.swift

```diff
@@ -3,6 +3,8 @@ import ComposableArchitecture
 import SwiftUI
 
 struct RootView: View {
+  @State var id = UUID()
```

Comment:
what about moving this down to the bottom of the struct so that it's not distracting? it's a hack after all 😬